draft-ietf-ace-oscore-profile.xml

```diff
@@ -652,7 +652,7 @@ OSCORE_Security_Context = {
           The client MUST generate a nonce value very unlikely to have been previously used with the same input keying material. This profile RECOMMENDS to use a 64-bit long random number as nonce's value. The client MUST store the nonce N1 as long as the response from the RS is not received and the access token related to it is still valid. The client MUST use CoAP and the Authorization Information resource as described in section 5.8.1 of <xref target="I-D.ietf-ace-oauth-authz"/> to transport the token and N1 to the RS.
         </t>
         <t>
-           Note that the use of the payload and the Content-Format is different from what described in section 5.8.1 of <xref target="I-D.ietf-ace-oauth-authz"/>, which only transports the token without any CBOR wrapping. In this profile, the client MUST wrap the token and N1 in a CBOR map. The client MUST use the Content-Format "application/ace+cbor" defined in section 8.14 of <xref target="I-D.ietf-ace-oauth-authz"/>. The client MUST include the access token using the correct CBOR label (e.g., "cwt" for CWT, "jwt" for JWT) and N1 using the 'nonce1' parameter defined in <xref target="nonce1"/>.
+           Note that the use of the payload and the Content-Format is different from what described in section 5.8.1 of <xref target="I-D.ietf-ace-oauth-authz"/>, which only transports the token without any CBOR wrapping. In this profile, the client MUST wrap the token and N1 in a CBOR map. The client MUST use the Content-Format "application/ace+cbor" defined in section 8.14 of <xref target="I-D.ietf-ace-oauth-authz"/>. The client MUST include the access token using the "access_token" parameter and N1 using the 'nonce1' parameter defined in <xref target="nonce1"/>.
```

Comment:
This one I'm not sure about.  The main thing I care about is consistency between this and the example, which does not use anything that I can see as being a "cwt" or "jwt" label.  I'm not sure what these labels would be or where the would be registered, either; combined with the fact that the RS probably has a good idea what kind of token its going to get anyway, this change seems more likely to be correct than any alternatives I can come up with.

Reply:
good catch! I remember making the change to the example but did not change here as well... Yes it should be access_token, as defined [in the framework](https://tools.ietf.org/html/draft-ietf-ace-oauth-authz-33#section-5.6.2).